### DISCUSSION TITLE
Machine does not move

### DISCUSSION THREAD

**Discussion by MrGadjit** [arkypita/LaserGRBL#2079] [General]
Hi 
Can anybody help me. I am using Laser Grbl with an Arduion based Control board. 

https://www.arduinoclub.shop/epages/83610251.sf/en_GB/?ViewObjectPath=%2FShops%2F83610251%2FProducts%2F7271255%2FSubProducts%2F7271255-0003

Iam switching between Estlcam and Grbl as I would like to use the machine as a CNC Milling machine still. What makes me wonder is that the Machine Operates perfectly fine using Estlcam but wont move as soon as I flash the Controlboard to GRBL useing LaserGrbl or lightburn. LaserGrbl recognizes the Controller but I don’t get any motion out of it.
I have installed everything but my Stepper motor don’t turn. I can feel that there are locked and they get warm which is perfectly fine but the don’t “Move”. 
Is there any parameter in the setting which disables the motors or something like that? I am using JMC IHSS57-36-20-31

Thanks 

Chris

**Comment by CJRN-cst** [comment]
Having same issue, using F-Yi GRBL Controller Board 3 Axis Controller Panel for DIY CNC 3018 1610 2418 board. LaserGRBL recognizes board.

**Comment by CJRN-cst** [comment]
Try setting $32 to 1, Laser mode, boolean, mine moved and just need to get travel direction correct
